packages/terra-worklist-data-grid/tests/jest/WorklistDataGrid.test.jsx

```diff
@@ -324,3 +337,473 @@ describe('Error handling - prop types', () => {
     expect(console.error).toHaveBeenCalledWith(expect.stringContaining(ERRORS.ROW_HEADER_INDEX_EXCEEDS_PINNED)); // eslint-disable-line no-console
   });
 });
+
+describe('Row selection', () => {
+  it('verifies only onRowSelect is called when space is used to select a row.', () => {
+    const mockOnCellSelect = jest.fn();
+    const mockOnRowSelect = jest.fn();
+    const mockOnEnableRowSelection = jest.fn();
```

Comment:
Should these be in some kind of `before` block since they are being declared for every test in this `describe`?

Reply:
+1

These should be in the `beforeAll()` as they don't need to be redefined in every test. We can use [mockClear](https://jestjs.io/docs/mock-function-api#mockfnmockclear) to reset the implementation between test runs. However we already have that option enabled in our [jest config](https://github.com/cerner/terra-toolkit/blob/fdb6c307005eb96fad4a9b468dd17f2012d69050/packages/jest-config-terra/jest.config.js#L6C1-L6C1) so we don't even need to do that!

Reply:
Since we created a `describe` for the rowSelection tests, we moved the mocks to `beforeEach` block instead.